tests/test_file_retriever.py

```diff
@@ -210,6 +205,10 @@ def test_get_web_w_auth(caplog, tmpdir, auth_configs, url, auth_type,
                 req_headers = m.request_history[0].headers
                 auth_header = req_headers.get('Authorization')
                 assert auth_header and 'Basic' in auth_header
+                username, password = base64.b64decode(
```

Comment:
Just wondering, does `requests_mock` always base64 encode the username, pw?

Reply:
It's not the mocker. RFC 2617 mandates base64 encoding. https://www.ietf.org/rfc/rfc2617.txt

Reply:
I know, but I was wondering whether `requests_mock` automatically does that for you or `requests` does it for you or if you as the developer must choose to follow RFC 2617 when sending the username, pw? Just wondering bc in our file retriever's web save method, we don't explicitly base64 encode username, pw and we prob should if its not done automatically.

Reply:
ok confirmed, requests does it for you